daprdocs/content/en/reference/api/scheduler_api.md

```diff
@@ -0,0 +1,121 @@
+---
+type: docs
+title: "Scheduler API reference"
+linkTitle: "Scheduler API"
+description: "Detailed documentation on the scheduler API"
+weight: 1300
+---
+
+{{% alert title="Note" color="primary" %}}
+Scheduler is currently in alpha.
+{{% /alert %}}
+
+With the Scheduler API, you can orchestrate jobs and tasks in your environment.
+
+## Schedule a job
+
+Schedule a job with the given name.
+
+```
+POST http://localhost:3500/v1.0-alpha1/job/schedule/<name>
+```
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`name` | Job name
+
+### Request content
+
+Any request content will be passed to the scheduler as input. The Dapr API passes the content as-is without attempting to interpret it.
+
+### HTTP response codes
+
+Code | Description
+---- | -----------
+`202`  | Accepted
+`400`  | Request was malformed
+`500`  | Request formatted correctly, error in dapr code or Scheduler control plane service
+
+### Response content
+
+The API call will provide a response similar to this:
+
+```json
```

Comment:
Can you turn this into an example, like has been done with the secrets here with a curl command and this response as the example https://docs.dapr.io/reference/api/secrets_api/#response-with-multiple-secrets-and-multiple-key--values-in-a-secret-eg-kubernetes